localstack/services/sts/sts_listener.py

```diff
@@ -0,0 +1,52 @@
+import re
+from requests.models import Request
+from localstack.constants import TEST_AWS_ACCOUNT_ID, MOTO_ACCOUNT_ID
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import to_str
+from localstack.services.generic_proxy import ProxyListener
+
+
+class ProxyListenerSTS(ProxyListener):
+    def forward_request(self, method, path, data, headers):
+        if method == 'POST' and path == '/':
+            data = self._reset_account_id(data)
+            return Request(data=data, headers=headers, method=method)
+
+        return True
+
+    def return_response(self, method, path, data, headers, response):
```

Comment:
I believe the 5 functions below have been copied from somewhere else. Let's please avoid copying code in general, to avoid having duplicate code in multiple places that becomes hard to maintain.

Can we create a small utility class, say, named `MessageConversion` in `utils/aws/aws_responses.py`, and add the methods below as static method definitions (`@staticmethod`) to the utility class (and then import them from the various places where we're using them)? That would be great, thanks.